src/upload/main.scss

```diff
@@ -275,9 +277,11 @@
 
         &-wrapper {
             position: relative;
+            box-sizing: content-box;
```

Comment:
I don't think we should use `box-sizing: content-box; `,  `border-box` is better, maybe we should ask designer for help